src/app/providers.tsx

```diff
@@ -23,22 +25,24 @@ export default function RootProviders ({
   );
 
   const BskyAuthProvider = dynamic(() => import('../components/BlueskyAuthProvider'), {
-    loading: () => <p>Loading...</p>,
+    loading: () => <Loader />,
     ssr: false
   })
 
   const StorachaAuthProvider = dynamic(() => import('../components/W3UIProvider'), {
-    loading: () => <p>Loading...</p>,
+    loading: () => <Loader />,
     ssr: false
   })
 
   return (
     <QueryClientProvider client={queryClient}>
-      <BskyAuthProvider>
-        <StorachaAuthProvider>
-          {children}
-        </StorachaAuthProvider>
-      </BskyAuthProvider>
+      <BackupsProvider>
+        <BskyAuthProvider>
+          <StorachaAuthProvider>
```

Comment:
I saw that you've updated this provider name like you mentioned in this comment. I called mine `StorachaProvider` &mdash; which I'll update now, to avoid merge conflicts when this PR gets into `main`.